index.js

```diff
@@ -1,7 +1,18 @@
-const emailLog = require('./email_log.json');
+const emailLog = require("./email_log.json");
 
 function countUniqueEmails(emailFile) {
   // do work here
+  const theList = emailFile.emails; //this is an array
+
+  return theList.reduce(function(p, c) {
```

Comment:
I would recommend using meaningful naming conventions instead of `p` and `c`. They should be named based on the value they represent.

Reply:
changes made, awaiting review.